WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormTableViewModel.swift

```diff
@@ -75,8 +75,8 @@ extension ProductFormSection.PrimaryFieldRow: Equatable {
         switch (lhs, rhs) {
         case (.images, .images):
             return true
-        case (let .name(name1), let .name(name2)):
-            return name1 == name2
+        case (let .name(name1, isEditable1), let .name(name2, isEditable2)):
+            return name1 == name2 && isEditable1 == isEditable2
```

Comment:
I think we could rely on the synthesized Equatable conformance, right?

Reply:
oh wow, I didn't realize that we don't need the custom `Equatable` implementation for an enum with associated values anymore! I couldn't find the reference where it's fixed in a Swift version, but will update to just have the form enums conform to `Equatable`

Reply:
updated here: https://github.com/woocommerce/woocommerce-ios/pull/2588/commits/31be2f9813645ff6329bbc19d157a1451ab43de9